packages/flutter/lib/src/services/autofill.dart

```diff
@@ -0,0 +1,287 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'text_input.dart';
+
+// TODO(LongCatIsLooong): expand the list to include every predefined value.
```

Comment:
We usually try to use issues in the TODO's. It makes it easier to track the missing pieces in the projects

Reply:
Added the full list of predefined hints.